<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">2048</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QWidget" name="mainWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QWidget" name="gameWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;2048&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="GameView" name="gameView"/>
          </item>
          <item>
           <widget class="GameInfoWidget" name="gameInfoWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="SettingsWidget" name="settingsTab">
             <attribute name="title">
              <string>Settings</string>
             </attribute>
            </widget>
            <widget class="LeaderboardWidget" name="leaderboardTab">
             <attribute name="title">
              <string>Leaderboard</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="GameStateWidget" name="gameStateWidget" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionFinnish"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInstruction"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="text">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="actionFinnish">
   <property name="text">
    <string notr="true">Suomi</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="text">
    <string>Instruction</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingsWidget</class>
   <extends>QWidget</extends>
   <header>widgets/settings/settingswidget.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeaderboardWidget</class>
   <extends>QWidget</extends>
   <header>widgets/leaderboard/leaderboardwidget.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameInfoWidget</class>
   <extends>QWidget</extends>
   <header>widgets/gameinfo/gameinfowidget.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameView</class>
   <extends>QGraphicsView</extends>
   <header>widgets/gameview/gameview.hh</header>
  </customwidget>
  <customwidget>
   <class>GameStateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/gamestate/gamestatewidget.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
